modular_citadel/code/modules/client/loadout/uniform.dm

```diff
@@ -89,3 +148,9 @@
 	category = slot_w_uniform
 	path = /obj/item/clothing/under/rank/head_of_security/skirt
 	restricted_roles = list("Head of Security")
+
+/datum/gear/turtleneck
+	name = "Tactitool Turtleneck"
+	category = slot_w_uniform
+	path = /obj/item/clothing/under/syndicate/tacticool
```

Comment:
That's 50% fire resistance and 40% acid resistance. Make a variant that doesn't have any protective qualities